CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,88 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
```

Comment:
```suggestion
- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
```

Reply:
Both should work, when it's a long sentence, I feel it's more readable to add `,` before and.

Reply:
Not strong opinion, just keep it.